sdk/formrecognizer/azure-ai-formrecognizer/src/test/java/com/azure/ai/formrecognizer/CopyAuthorizationSerializerTest.java

```diff
@@ -9,19 +9,19 @@
 import java.io.IOException;
 
 import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertFalse;
-import static org.junit.jupiter.api.Assertions.assertNull;
 import static org.junit.jupiter.api.Assertions.assertTrue;
 
 public class CopyAuthorizationSerializerTest {
 
     private static final String MODEL_ID = "97bf97bc-3210-4c84-a0ef-9472dec711a0";
     private static final String ACCESS_TOKEN = "769cd41c-9602-4373-ba38-a2531643e041";
-    private static final String RESOURCE_ID = "77bf97bc-3210-4c84-a0ef-9472dec712a0";
+    private static final String RESOURCE_ID = "77bf97bc-3210-4c84-a0ef-9462dec712a0";
     private static final String REGION = "westus2";
     private static final long EXPIRES_ON = 637258696186131476L;
     private static final String EXPECTED_JSON_STRING = "{\"modelId\":\"97bf97bc-3210-4c84-a0ef-9472dec711a0\","
-        + "\"accessToken\":\"769cd41c-9602-4373-ba38-a2531643e041\",\"expirationDateTimeTicks\":637258696186131476}";
+        + "\"accessToken\":\"769cd41c-9602-4373-ba38-a2531643e041\","
```

Comment:
is this a fake value? just making sure we are not exposing a real access token

Reply:
Yup, it's fake. Just wanted to use the correct format.